shell.nix

```diff
@@ -12,7 +12,7 @@ pkgs.mkShell {
     which # 2.21
     rustc # 1.63.0
   ] ++ lib.optionals stdenv.isDarwin [ 
-    libiconv
+    libiconv 
```

Comment:
Unnecessary space.

Reply:
Addressed in [4512088](https://github.com/status-im/nwaku/pull/1240/commits/45120888440c9264b1952fa8ecb081be47c1b548)